homeassistant/components/ziggo_next/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""The Ziggo Next integration."""
+import asyncio
+import logging
+
+import voluptuous as vol
+from ziggonext import ZiggoNext
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import CONF_COUNTRY_CODE, DOMAIN, ZIGGO_API
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
```

Comment:
This needs to have the schema written out

See `august/__init__.py` CONFIG_SCHEMA